File: kyu8/string-repeat.php
<?php
/*
DESCRIPTION:
Write a function that accepts an integer n and a string s as parameters, and returns a string of s repeated exactly n times.

Examples (input -> output)
6, "I"     -> "IIIIII"
5, "Hello" -> "HelloHelloHelloHelloHello"
*/
function repeatStr($n, $str)
  {
    $a = NULL;
    for($i = 0; $i < $n;$i++){
      $a .= $str;
    }
    return $a;
  }

File: kyu8/string-repeat.swift
/*
Write a function that accepts an integer n and a string s as parameters, and returns a string of s repeated exactly n times.

Examples (input -> output)
6, "I"     -> "IIIIII"
5, "Hello" -> "HelloHelloHelloHelloHello"
*/

func repeatStr(_ n: Int, _ string: String) -> String {
  // Code here:
  var a = ""
  for i in 0..<n {
    a += "\(string)"
	}
  return a
}